### ISSUE TITLE
Provide a command line switch for choosing terminal profile

### ISSUE THREAD

**Issue by GoogleCodeExporter** [brockgr/csshx#33]
```
What steps will reproduce the problem?
1. Open Terminal Preferences to the Advanced Tab
2. Ensure the "Audible Bell" is enabled on the profile you use
3. Use csshX to launch a reasonable number of connections (say, 6)
4. Press tab twice or generally cause a bell to occur in the shell
5. I hope you had your speakers/headphones turned down


What is the expected output? What do you see instead?

The system default sound will be LOUD. For whatever reason, the sound being 
played at the same time starts increasingly the overall volume on it.

I have an 8 machine cluster that I work on regularly, and when I cause a system 
bell to occur, I usually rip off my headphones in agony because the "Morse" OSX 
system sound is now ringing constantly through my head.

The resolution I'd like to have is to create a clone of my profile, and likely 
name it csshX, and on csshX invocation add in --profile csshX, disabling the 
audible bell (and likely enabling the visual bell).

Perhaps this could be added as a "slave_profile" property to csshrc as well?
That or a simple: 'alias csshX="csshX --profile csshX"' is in my future.


What version of csshX (do a "csshX -v") are you using? On what operating
system version?

csshX v0.73-168
OSX 10.6.4


Please provide any additional information below.

I'm open to other methods for accomplishing this. Let's brainstorm!
```

Original issue reported on code.google.com by `vxjaso...@gmail.com` on 29 Sep 2010 at 5:46

**Comment by GoogleCodeExporter** [comment]
```
csshX morse - LOL

Ok - just committed to the SVN version - try this:

       −−slave_settings_set, −−sss string
           Change the "settings set" for slave windows.

       −−master_settings_set, −−mss string
           Change the "settings set" for master windows.

Naturally, you can put these in the csshrc too.

```

Original comment by `gavin.brock` on 30 Sep 2010 at 2:38
- Changed state: **Fixed**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Awesome, Gavin. Your response time, quality, and support are absolutely superb. 
I love this project.
```

Original comment by `vxjaso...@gmail.com` on 1 Oct 2010 at 4:39

**Comment by GoogleCodeExporter** [comment]
```
Released in 0.73
```

Original comment by `gavin.brock` on 8 Oct 2010 at 1:27
- Changed state: **Done**
